kytos/core/tcp_server.py

```diff
@@ -98,8 +102,10 @@ def setup(self):
         if server_port in self.known_ports:
             protocol_name = self.known_ports[server_port]
         else:
-            protocol_name = f'{server_port:04d}'
```

Comment:
@viniciusarcanjo I would leave this as it is over here to allow the TCPServer  to work on any port even with an unkown southbound protocol.
I think it would be better to 'update' the 'known_ports' dictionary by doing, on the `KytosRequestHandler` `setup()` the following:

```python
def setup(self):
    self.addr = self.client_address[0]
    self.port = self.client_address[1]
    if self.server.protocol_name:
        self.known_ports.update((self.port, self.server.protocol_name))
```
Then the rest of the code can remain the same with the right fallback to using the port number on the event name if we do not know the protocol that is being used.

Reply:
Good suggestion! I'll do this, except that this self.port is the port of the client (that came from this request), I believe you meant the port of the server, i.e., `server_port = self.server.server_address[1]`

Reply:
Yes, you are right!! =)